.github/workflows/smoke-test.yml

```diff
@@ -21,12 +21,16 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Install the package
       run: |
         sudo apt-get update
         python -m pip install --upgrade pip
-        pip install .
-        pip install .[dev]
+        pip install -e ".[dev]"
     - name: Run unit tests with pytest
       run: |
         python -m pytest tests
```

Comment:
Could you make this verbose so it output test by test status? simply add `-vvv`

Reply:
@lsetiawan do you think this is something that should go in the template?

Reply:
I normally like a more verbose test output, but this does make the report out longer. Once you try it, let me know what you prefer. I try to follow the Python's mantra of more explicit rather than implicit 😛